### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by trodge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
 
+FEATURES:
+* **New Resource:** `google_cloudbuildv2_connection` (ga) ([#5857](https://github.com/hashicorp/terraform-provider-google-beta/pull/5857))
+* **New Resource:** `google_cloudbuildv2_repository` (ga) ([#5857](https://github.com/hashicorp/terraform-provider-google-beta/pull/5857))
+* **New Resource:** `google_gkeonprem_bare_metal_admin_cluster` ([#5858](https://github.com/hashicorp/terraform-provider-google-beta/pull/5858))
+* **New Resource:** `google_network_security_address_group` (ga) ([#5865](https://github.com/hashicorp/terraform-provider-google-beta/pull/5865))
+* **New Resource:** `google_network_security_gateway_security_policy_rule` (ga) ([#5866](https://github.com/hashicorp/terraform-provider-google-beta/pull/5866))
+* **New Resource:** `google_network_security_gateway_security_policy` (ga) ([#5866](https://github.com/hashicorp/terraform-provider-google-beta/pull/5866))
+* **New Resource:** `google_network_security_url_lists` (ga) ([#5866](https://github.com/hashicorp/terraform-provider-google-beta/pull/5866))
+* **New Resource:** `google_network_services_gateway` (ga) ([#5866](https://github.com/hashicorp/terraform-provider-google-beta/pull/5866))
```
Comment (shuyama1) [inline CHANGELOG.md:11]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+
+IMPROVEMENTS:
+
+* bigquery: added `storage_billing_model` argument to `google_bigquery_dataset` ([#5868](https://github.com/hashicorp/terraform-provider-google-beta/pull/5868))
+* bigquery: added `external_data_configuration.metadata_cache_mode` and `external_data_configuration.object_metadata` to `google_bigquery_table` ([#5856](https://github.com/hashicorp/terraform-provider-google-beta/pull/5856))
+* bigquery: made `external_data_configuration.source_fomat` optional in `google_bigquery_table` ([#5856](https://github.com/hashicorp/terraform-provider-google-beta/pull/5856))
+* certificatemanager: added `issuance_config` field to `google_certificate_manager_certificate` resource ([#5860](https://github.com/hashicorp/terraform-provider-google-beta/pull/5860))
+* cloudbuild: added `repository_event_config` field to `google_cloudbuild_trigger` resource (ga) ([#5857](https://github.com/hashicorp/terraform-provider-google-beta/pull/5857))
+* cloudbuildv2: added `gitlab_config` field to `google_cloudbuildv2_connection` resource (beta) ([#5848](https://github.com/hashicorp/terraform-provider-google-beta/pull/5848))
```
Comment (shuyama1) [inline CHANGELOG.md:20]:
```suggestion
* cloudbuildv2: added `gitlab_config` field to `google_cloudbuildv2_connection` resource ([#5848](https://github.com/hashicorp/terraform-provider-google-beta/pull/5848))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+* compute: added field `http_keep_alive_timeout_sec` to resource `google_compute_target_http_proxy` ([#5864](https://github.com/hashicorp/terraform-provider-google-beta/pull/5864))
+* compute: added field `http_keep_alive_timeout_sec` to resource `google_compute_target_https_proxy` ([#5864](https://github.com/hashicorp/terraform-provider-google-beta/pull/5864))
+* compute: added support for updating labels in `google_compute_external_vpn_gateway` ([#5875](https://github.com/hashicorp/terraform-provider-google-beta/pull/5875))
+* container: added field `tpu_topology` under `placement_policy` in resource `google_container_node_pool` ([#5871](https://github.com/hashicorp/terraform-provider-google-beta/pull/5871))
+* container: made `monitoring_config.enable_components` optional in `google_container_cluster` ([#5872](https://github.com/hashicorp/terraform-provider-google-beta/pull/5872))
```
Comment (shuyama1) [inline CHANGELOG.md:25]:
```suggestion
```
This seems a ga-only change

**Review by shuyama1 — DISMISSED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+* cloudbuildv2: added `gitlab_config` field to `google_cloudbuildv2_connection` resource ([#5848](https://github.com/hashicorp/terraform-provider-google-beta/pull/5848))
+* compute: added field `http_keep_alive_timeout_sec` to resource `google_compute_target_http_proxy` ([#5864](https://github.com/hashicorp/terraform-provider-google-beta/pull/5864))
+* compute: added field `http_keep_alive_timeout_sec` to resource `google_compute_target_https_proxy` ([#5864](https://github.com/hashicorp/terraform-provider-google-beta/pull/5864))
+* compute: added support for updating labels in `google_compute_external_vpn_gateway` ([#5875](https://github.com/hashicorp/terraform-provider-google-beta/pull/5875))
+* container: added field `tpu_topology` under `placement_policy` in resource `google_container_node_pool` ([#5871](https://github.com/hashicorp/terraform-provider-google-beta/pull/5871))
+* gkehub: promoted the `google_gke_hub_feature` resource's `fleetobservability` block to GA. ([#5861](https://github.com/hashicorp/terraform-provider-google-beta/pull/5861))
```
Comment (melinath) [inline CHANGELOG.md:14]: GA-only
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+* iamworkforcepool: added `oidc.client_secret` field to `google_iam_workforce_pool_provider` and new enum values `CODE` and `MERGE_ID_TOKEN_OVER_USER_INFO_CLAIMS` to `oidc.web_sso_config.response_type` and `oidc.web_sso_config.assertion_claims_behavior` respectively ([#5853](https://github.com/hashicorp/terraform-provider-google-beta/pull/5853))
+* sql: added `settings.data_cache_config` to `sql_database_instance` resource. ([#5869](https://github.com/hashicorp/terraform-provider-google-beta/pull/5869))
+* sql: added `settings.edition` field to `sql_database_instance` resource. ([#5869](https://github.com/hashicorp/terraform-provider-google-beta/pull/5869))
+* vertexai: supported`shard_size` in `google_vertex_ai_index` ([#5874](https://github.com/hashicorp/terraform-provider-google-beta/pull/5874))
+
+BUG FIXES:
+* compute: made `google_compute_router_peer.peer_ip_address` optional ([#5855](https://github.com/hashicorp/terraform-provider-google-beta/pull/5855))
+* monitoring: fixed an issue causing `google_monitoring_monitored_project` to appear to be deleted ([#5876](https://github.com/hashicorp/terraform-provider-google-beta/pull/5876))
```
Comment (melinath) [inline CHANGELOG.md:22]:
```suggestion
```

fixed in patch release

**Review by melinath — APPROVED** [review]

**Pull request merged**